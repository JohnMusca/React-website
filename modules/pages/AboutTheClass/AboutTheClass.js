import React from 'react'
import Header from '../../common/Header/Header'
import Body from '../../common/Body/Body'
import Footer from '../../common/Footer/Footer'

//Content passed into body component
var content = "<p>The Buk Sing Choy Lay Fut Kung Fu School services the Northern and Western suburbs and is located at 74 Melville Road, Brunswick West, which has immediate access to the suburbs of; Strathmore, Essendon, Pascoe Vale, Moonee Ponds, Ascot Vale, Maribyrnong, Kensington, Flemington, North Melbourne, Brunswick and Coburg.</p>" +
              "<p>Training times are Tuesday and Thursday - 6:30pm to 8:30pm.</p>" +
              "<p>Classes cater to all age groups, genders and backgrounds.</p>" +
              "<p>Small class sizes provide a high degree of pastoral care and teaching is conducted in a non violent non threatening manner with particular regard to humility and respect.</p>" +
              "<p>Dress consists of: gym shoes, T Shirt (black preferable) and long Black Martial Arts pants (shorts are acceptable initially).</p>" +
              "<p>Cost to be instructed in Buk Sing Choy Lay Fut Kung Fu comprises:" +
              "<ul>" +
                "<li>Joining fee which includes cost of Buk Sing Choy Lay Fut training T Shirt;</li>" +
                "<li>Monthly instruction fee;</li>" +
                "<li>No grading or other ongoing fees.</li>" +
              "</ul></p>" +
              "<p>Sifu Gary Mallia welcomes you to your Free Introductory Class and looks forward to assisting with what is considered a significant life changing experience.</p>";

/**
* About the class page component used to render the page.
*
* Props:      None
* 
* State vars: None
*/
class AboutTheClass extends React.Component {
  render () {
    return (
      <div>
        <div  class="hero-image-about"></div>
        <Body heading="About the class" content={content} />
        <Footer />
      </div>
    )
  }
}

export default AboutTheClass;
