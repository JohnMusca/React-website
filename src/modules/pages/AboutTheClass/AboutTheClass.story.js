import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'
import AboutTheClass from './AboutTheClass'

storiesOf('Pages', module)
  .add('AboutTheClass', () => (
      <AboutTheClass />
  ));
