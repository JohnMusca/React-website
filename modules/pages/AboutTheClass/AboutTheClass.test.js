import React from 'react';
import AboutTheClass from './AboutTheClass';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('AboutTheClass component', () => {
  const component = renderer.create(
		<AboutTheClass />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
