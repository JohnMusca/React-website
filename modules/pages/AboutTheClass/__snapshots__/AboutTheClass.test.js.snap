// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutTheClass component 1`] = `
<div>
  <div>
    <p>
      About the class content
    </p>
  </div>
  <div>
    <p>
      Footer
    </p>
  </div>
</div>
`;
